liquidation-monitor/src/agent.js

```diff
@@ -0,0 +1,522 @@
+const {
+  Finding,
+  FindingSeverity,
+  FindingType,
+  ethers,
+  getEthersProvider,
+} = require('forta-agent');
+const BigNumber = require('bignumber.js');
+
+// To-do: Replace node-fetch with axios and remove TS
```

Comment:
Do what the `To-do` states and remove comment.

Reply:
Removed comments